autometa/taxonomy/ncbi.py

```diff
@@ -299,23 +281,19 @@ def parse_names(self):
 
     # @timeit
     def parse_nodes(self):
-        """Short summary.
+        """
+        Parses through `nodes.dmp` database and returns a dictionary with child, parent and rank
 
         Parameters
         ----------
-        verbose : boolean
+        self.verbose : boolean
             prints progress to terminal (the default is False).
```

Comment:
Similarly here, there are no parameters to be provided in the method, so you should remove this from the docstring